openstack_api_reporting.py

```diff
@@ -187,19 +187,17 @@ def decodeIDs(config, events):
     for domain in domains:
```

Comment:
we could probably move this down into the other for domain in domains loop as well.